src/AiidaDFTK.jl

```diff
@@ -101,7 +102,36 @@ Run a DFTK calculation from a json input file.
 Output is by default written to `stdout` and `stderr`.
 The list of generated output files is returned.
 """
-function run_json(filename::AbstractString; extra_output_files=String[])
+function run_json(filename::AbstractString; extra_output_files=String[], min_version=typemin(VersionNumber), max_version=typemax(VersionNumber))
+    # We write a few key messages to an error file that we can parse from
+    # the AiiDA plugin to detect common failure modes.
+    errorfile = "errors.log"
+    if isfile(errorfile)
+        error("Error file $errorfile already exists!")
+    end
+    push!(extra_output_files, errorfile)
+    function report_error(msg)
+        mpi_master() || return
+        open(errorfile, "a") do io
+            println(io, msg)
+            flush(io)
+        end
+        println(msg) # also print to stdout
+    end
+    function fatal_error(msg)
+        report_error(msg)
+        error(msg)
+    end
+
+    report_error("Imports succeeded -- This indicates that AiidaDFTK was installed correctly and that the MPI environment is likely correct.")
```

Comment:
Line break. If you want we can actually setup automatic source code formatting for this repo. I don't think here we will have many places where the formatting really matters from a scientific point of view.

Reply:
I doubt it would break such a long string apart.

Reply:
if you give it a maximum character count, yes.

Reply:
```
julia> format_text("""x = "here is a veeeeeeeeeeery long string let's see if the formatter wants to do something about it or not. Personally, I have my doubts but let's try it anyway. If it does not work I can try to reduce the number of allowed characters in each line or something.";""")
"x = \"here is a veeeeeeeeeeery long string let's see if the formatter wants to do something about it or not. Personally, I have my doubts but let's try it anyway. If it does not work I can try to reduce the number of allowed characters in each line or something.\";"
```
Doesn't seem to

Reply:
even if you configure the maximal line number count ? I'm not sure it does it by default.